maintainers/maintainer-list.nix

```diff
@@ -4414,6 +4414,12 @@
     githubId = 524268;
     name = "Koral";
   };
+  koslambrou = {
+    email = "koslambrou@gmail.com";
+    github = "koslambrou";
+    githubId = 2037002;
+    name = "Konstantinos";
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
a commit in this PR or in a new PR ?